### PR TITLE
772041

### LINKED ISSUE #3260: 772041
GDoc: 

Category: PCR/qPCR prep

### PR DESCRIPTION
**Pull request by eskila**
This protocol is ready to review, but I need some more info from the customer before closing this PR.

Closes #3260

### CODE REVIEW

**Review by chazmanc — COMMENTED** [review]
Great starting point - please tag me after incorporating the details

**Comment by chazmanc** [comment]
Also added the *work in progress* (WIP) label

**Comment by eskila** [comment]
I think this protocol is ready enough for an initial submission, and when I get more details on what other labware they want to use for their destination plate options I can add those as well.

File: protocols/772041/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+* PCR
+	* PCR Prep
+
+## Description
+This protocol distributes PCR mastermix from a source NEST 12 well reservoir to a target Applied Biosystems Microamp optical 384 well 30 µL plate (or other PCR plate) with an 20 µL 8-channel pipette loaded in the left mount. The user can choose how many wells to transfer to (rounding up to the nearest number of columns) as well as how much volume to add to each well. The user can also choose whether they want to reuse tips, or pick up new tips for each transfer.
+
+Explanation of parameters below:
+* `Number of wells to distribute mastermix to`: How many wells to transfer mastermix to on the target plate. This will be rounded up to the nearest number of full columns.
+* `Volume of mastermix`: Volume of mastermix to add to each well (µL)
+* `Reuse tips?`: Yes: Reuse tips when distributing the mastermix. No: Drop the tips after each distribution, and pick up new tips before each aspiration.ß
```
Comment (chazmanc) [inline protocols/772041/README.md:16]: weird element at the end of this line

File: protocols/772041/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+---
+
+### Labware
+* [NEST 12-Well Reservoirs, 15 mL](https://shop.opentrons.com/nest-12-well-reservoirs-15-ml/)
+* [MicroAmp Optical 384-Well Reaction Plate with barcode](https://www.thermofisher.com/order/catalog/product/4309849)
+
+### Pipettes
+* [20 µL multi-channel pipette](https://shop.opentrons.com/8-channel-electronic-pipette/)
+* [20 µL filter tips](https://shop.opentrons.com/opentrons-20ul-filter-tips/)
```
Comment (chazmanc) [inline protocols/772041/README.md:27]: let's add this to ### labware

**Pull request merged**